src/library.js

```diff
@@ -3191,8 +3191,8 @@ mergeInto(LibraryManager.library, {
   $getExecutableName: function() {
 #if MINIMAL_RUNTIME // MINIMAL_RUNTIME does not have a global runtime variable thisProgram
 #if ENVIRONMENT_MAY_BE_NODE
-    if (ENVIRONMENT_IS_NODE && process['argv'].length > 1) {
-      return process['argv'][1].replace(/\\/g, '/');
+    if (ENVIRONMENT_IS_NODE && process.argv.length > 1) {
+      return process.argv[1].replace(/\\/g, '/');
```

Comment:
I also don't see `argv` in the externs, so what prevents it from being minified?

Reply:
I think closure itself ships with it own internal set of externs for node: https://github.com/google/closure-compiler/blob/ce52aa6c45fb14a6d662ecf312b660a289f12906/contrib/nodejs/globals.js#L124

You can see this file in `node_modules/google-closure-compiler/contrib/nodejs/globals.js` under you emscripten directory.